text/java/0003-clojure-tools.md

```diff
@@ -0,0 +1,33 @@
+# Clojure Tools Buildpack
+
+## Summary
+
+A Clojure Tools buildpack for building Clojure applications using [deps][d] and [tools build][t] has been created [by the community][b].  It is based on the same technologies and is nearly indistinguishable from the other Paketo Buildpacks, especially the JVM build-system buildpacks, and it should be adopted into the project.
+
+[d]: https://clojure.org/guides/deps_and_cli
+[t]: https://clojure.org/guides/tools_build
+[b]: https://github.com/eddumelendez/clojure
+
+## Motivation
+
+This is an additional buildpack, for building JVM-based applications based on Clojure, and offers yet another workload that can be built by Paketo buildpacks.
+
+## Detailed Explanation
+
+N/A
```

Comment:
Can you expand on what needs to be changed to transfer ownership to Paketo? Either in Details Explanation or under Implementation. See [here](https://github.com/paketo-buildpacks/rfcs/blob/4ad0d65ef1662fb5de2871e64013deddc4f529f4/text/java/0001-microsoft-jdk.md#implementation). It should be very similar to that section.